### PR TITLE
Add PHP7 to travis build

### PR DESCRIPTION
**Pull request by dakota**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - '7'
```
Comment (lorenzo) [inline .travis.yml]: in qoutes?

**Comment by markstory** [comment]
wooo

File: .travis.yml (hunk 1 above)
Comment (ADmad) [inline .travis.yml]: @lorenzo That's what travis [docs](http://docs.travis-ci.com/user/languages/php/#PHP-7) show.

**Comment by dereuromark** [comment]
Fancy :D

**Comment by AD7six** [comment]
> Tests: 5627, Assertions: 16697, Failures: 29, Errors: 25, Skipped: 221.
> Segmentation fault (core dumped)

Seems to work fine :+1:

**Comment by ADmad** [comment]
It needs to be added to allow_failures until all errors can be sorted out.

**Comment by dakota** [comment]
Added to allow failures.

**Comment by jrbasso** [comment]
Just for a note, I tested PHP 7 with Cake 3 and found some issues. I sent it to internal and they fixed really quick.

Most of the remaining issues seems to be 2 things. One is something on the Response class and something else in some memory leak or overflow in PHP core, but I couldn't narrow down to the exactly issue yet and how to reproduce in an isolated way.

**Comment by markstory** [comment]
Nice work @jrbasso :+1:

**Comment by ADmad** [comment]
Great @jrbasso

**Pull request merged**